configure.ac

```diff
@@ -1310,11 +1310,11 @@ dnl python checks
 dnl ***************************************************************************
 if test "x$enable_python" != "xno"; then
   if test "x$with_python" = "xauto"; then
-    PKG_CHECK_MODULES(PYTHON, python3 >= 3, python_found="yes", python_found="no")
+    PKG_CHECK_MODULES(PYTHON, python >= 3, python_found="yes", python_found="no")
```

Comment:
I suggest using `python3` for checking Python 3 existence and use `python` for test Python 2.

The [Ubuntu documentation](https://wiki.ubuntu.com/Python/3) says that making Python 3 the default **does not mean**:
- `/usr/bin/python` will point to Python 3. **No, this is not going to happen** (unless [PEP 394](https://www.python.org/dev/peps/pep-0394/) advocates otherwise, which is doubtful for the foreseeable future). `/usr/bin/python` and `/usr/bin/python2` will point to Python 2.7 and `/usr/bin/python3` will point to the latest supported Python 3 version.

Reply:
`PKG_CHECK_MODULES` checks for the name of the corresponding `.pc`, not for the binary.
This can vary on different platform (for example: Arch Linux uses `python2` and `python3`, Ubuntu has a `python` alias for `python2`. I would check both `python{2,3}` and `python` as well.


Arch:
```
$ pkg-config --modversion python
Package python was not found in the pkg-config search path. 

$ pkg-config --modversion python3
3.6
```

Reply:
Thank you both for your comments. I updated accordingly, both ubuntu and arch is supported now.